app/src/test/java/com/codesoom/assignment/controllers/UserControllerTest.java

```diff
@@ -0,0 +1,141 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.application.UserCommandService;
+import com.codesoom.assignment.dto.UserRequest;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+class UserControllerTest {
+    @Autowired
+    private UserCommandService userCommandService;
+    @Autowired
+    private MockMvc mockMvc;
+
+    private ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
objectMapper가 재할당된다면 곤란하지 않을까요?
```suggestion
    private final ObjectMapper objectMapper = new ObjectMapper();
```

Reply:
80ddb675bbddb5694eaabf7ebc0c8441b921f296
final 추가하였습니다!